src/PublishPort.jsx

```diff
@@ -63,7 +63,7 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
             <FormGroup className="pf-m-2-col-on-md"
                         label={_("Protocol")}
                         fieldId={id + "-protocol"}>
-                <FormSelect className='pf-c-form-control container-port-protocol'
+                <FormSelect className='pf-v5-c-form-control container-port-protocol'
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230601-052642-7ca6cf83-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)